### ISSUE TITLE
"Hosted by GBIF" footer bar for hosted portals

### ISSUE THREAD

**Issue by javiesgm** [gbif/hosted-portals#222]
It is necessary to include a footer bar with a "Hosted by GBIF" sign
The bar should be dark grey #1A1A1A and 70px height. 

Here is the .svg
![hosted-by-gbif](https://user-images.githubusercontent.com/14220404/159012460-871613f1-a7ee-4022-b091-fba249e2f45e.svg)

And here is a mockup
![hosted-by-70px](https://user-images.githubusercontent.com/14220404/159012591-e51ec0b1-8bb6-4c9f-a0b1-20f18ff0f624.jpg)

It should include a link to a GBIF.org page, but I'm still working on it. In the meantime, please link to this news item:
https://www.gbif.org/news/6RG9WSU3knIxyhcDUBQTJY/first-five-hosted-portals-emerge-from-gbif-pilot-programme

**Comment by MortenHofft** [comment]
To be shown in production, the sites needs to get a new release. For some sites that could take months. The alternative is that we create a nonsense release for each site. Or better - that we add a way to do a production deployment from Jenkins (or even just as a script)

**Comment by MattBlissett** [comment]
This is one option, but it releases the most recent commit which isn't necessarily what we want (e.g. most recent commit of PBIF is broken), other sites might have commits that aren't ready for release.

```
curl -X POST -d 'payload={"action":"published"}' -i 'https://builds.gbif.org/job/hp-north-america/buildWithParameters?token=XXX'
```

(Or with `{"action":"published"}` pasted into "Build with parameters" in Jenkins.)

**Comment by MortenHofft** [comment]
Exactly - what I want is to rebuild and deploy the last release. In the past I have checked out the last release commit and then released that anew (I think that was the procedure), but it is fairly time consuming and awkward.

**Comment by MortenHofft** [comment]
Hmm - I've tried the approach with going back to past jobs in Jenkins and cp the payload. That worked in the sense that they all got the new footer.

BUT, as I did it for sprep it  failed. Which makes me think that it isn't doing what I thought it would do - namely use the tag to deploy a new version. But simply takes head. Else I do not see why SPREP would fail. In that case it means that I have most likely deployed head for all the sites. Which wasn't what I wanted at all.
